### DISCUSSION TITLE
A “problem” about ui.notify() display...

### DISCUSSION THREAD

**Discussion by zhangguolin333** [zauberzeug/nicegui#3716] [Q&A]
```py
@ui.page('/pick_a_file', response_timeout=10)
async def pick_a_file_page():
    files = await app.native.main_window.create_file_dialog(allow_multiple=False)
    if not files:
        ui.notify('not select a file', position='center', type='warning')
        return
    file_path = files[0]
    notification = ui.notify('file is being imported...', position='center', type='ongoing', timeout=0)
    await io_bound(lambda: get_db_all_datas(file_path))
    notification.close()
    ui.notify('imported successful', position='center', type='positive')
```

In my example, **"get_db_all_datas"** is a time-consuming operation. I want to display a notify saying **"file is being imported..."** before it be called, and then show **"imported successful”** after the operation is completed. The current issue is that the prompt **“file is being imported...”** does not appear before **“get_db_all_datas”** starts executing, and both **“file is being imported...”** and **“imported successful”** appear simultaneously after the execution is completed. This is not what I expect. How can I achieve the desired behavior?

**Comment by python-and-novella** [comment]
When you're asking for help, it is important to provide a [minimal executable example (MRE)](https://en.wikipedia.org/wiki/Minimal_reproducible_example), if possible.
ui.notification | NiceGUI
https://nicegui.io/documentation/notification
```python3
from nicegui import ui
import asyncio

async def test():
    notification = ui.notification('file is being imported...', position='center', type='ongoing', timeout=0)
    await asyncio.sleep(5)
    notification.dismiss()
    ui.notify('imported successful', position='center', type='positive')

@ui.page('/')
async def pick_a_file_page():
    ui.button('test',on_click=test)

ui.run(native=True)
```

**Reply by zhangguolin333** [reply]
ok,  thanks!~~
